web/src/components/Home/RedwoodIs/RedwoodIs.tsx

```diff
@@ -37,10 +37,7 @@ const RedwoodIs = () => {
           Get Started on our Established Base
         </a>
         <div className="flex h-[53px] items-center justify-between gap-7 rounded-[4px] border-2 border-maiTai px-7">
-          <div
-            contentEditable={true}
```

Comment:
@ahaywood Was there any reason for this to be contentEditable? It was making react produce warnings on the terminal so I removed it because I thought we didn't need it.